### PR TITLE
docs: run a specific test

### PR DESCRIPTION
**Pull request by WenyXu**

### CODE REVIEW

File: docs/developer-guide/tests/sqlness-test.md [hunk 1]
```diff
@@ -38,4 +38,13 @@ cargo run --bin sqlness-runner
...
\ No newline at end of file
+If not, congratulations, the test is passed 🥳!
+
+### Run a specific test
+
+```shell
+cargo run --bin sqlness-runner your_test
+```
+
+If the second argument was specified, only run cases containing the second argument string in their names.
```
Comment (xtang) [inline docs/developer-guide/tests/sqlness-test.md:49]:
```suggestion
If you specify a second argument, only test cases containing the specified string in their names will be executed.
```

**Comment by xtang** [comment]
@WenyXu Thanks for contributing to the docs. @evenyag Please take a look.

File: docs/developer-guide/tests/sqlness-test.md [hunk 2]
```diff
@@ -38,4 +38,13 @@ cargo run --bin sqlness-runner
...
\ No newline at end of file
+If not, congratulations, the test is passed 🥳!
+
+### Run a specific test
+
+```shell
+cargo run --bin sqlness-runner your_test
+```
+
+If you specify a second argument, only test cases containing the specified string in their names will be executed.
```
Comment (waynexia) [inline docs/developer-guide/tests/sqlness-test.md:49]: and sqlness also support filter on env. Filter is accepted as regex string. And case name will be examine in form of `env:case`.

Comment (xtang) [reply docs/developer-guide/tests/sqlness-test.md:49]: @waynexia, do we need to include this statement in the doc?

**Review by xtang — APPROVED** [review]

File: docs/developer-guide/tests/sqlness-test.md (hunk 2 above)
Comment (waynexia) [reply docs/developer-guide/tests/sqlness-test.md:49]: Yes, please. I rephrase it to:

sqlness also supports filtering based on environment. The filter is accepted as a regex string and the case name will be examined in the format of 'env:case'.

File: docs/developer-guide/tests/sqlness-test.md [hunk 3]
```diff
@@ -38,4 +38,13 @@ cargo run --bin sqlness-runner
...
\ No newline at end of file
+If not, congratulations, the test is passed 🥳!
+
+### Run a specific test
+
+```shell
+cargo sqlness your_test
+```
+
+If you specify a second argument, only test cases containing the specified string in their names will be executed.
```
Comment (xtang) [inline docs/developer-guide/tests/sqlness-test.md:49]:
```suggestion
If you specify a second argument, only test cases containing the specified string in their names will be executed. Sqlness also supports filtering based on environment. The filter is accepted as a regex string and the case name will be examined in the format of `env:case`.
```
